app/controllers/hostgroups_controller.rb

```diff
@@ -79,9 +79,9 @@ def destroy
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
+      hostgroup = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id'])
       render :partial => "common_parameters/inherited_parameters",
-             :locals => { :inherited_parameters => inherited_parameters }
+             :locals => { :inherited_parameters => hostgroup.parent_params(true), :parameters => hostgroup.group_parameters }
```

Comment:
`inherited_params` returns also the related os and domain parameters and hostgroup wasn't actually showing them.
We can add all the inherited parameters instead of just the parent group parameters but I wanted to preserve the current situation.
I don't mind changing this.